types/api/v1beta1/event.proto

```diff
@@ -0,0 +1,124 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+import "google/protobuf/timestamp.proto";
+
+
+message Event {
+    google.protobuf.Timestamp timestamp = 1;
+    uint32 id = 2;
+    string name = 3;
+    Policies policies = 4;
+    Context context = 5;
+    // repeated EventData data = 6; TODO: event data is coming on a separate PR
+    optional Threat threat = 7;
+}
+
+message Policies {
+    repeated string matched = 1;
+}
+
+message Context {
+    optional Process process = 1;
+    optional Container container = 2;
+    optional K8S k8s = 3;
+}
+
+message Process  {
+    Executable executable = 1;
+    int32 pid = 2;
+    int32 namespaced_pid = 3;
+    uint32 entity_id = 4;
+    User real_user = 5;
+    Thread thread = 6;
+    optional Parent parent = 7;
+}
+
+message Parent  {
+    Executable executable = 1;
+    int32 pid = 2;
+    optional int32 namespaced_pid = 3;
+    uint32 entity_id = 4;
+    optional User real_user = 5;
+    optional Thread thread = 6;
+    optional Process parent = 7;
+}
+
+message Executable {
+    string name = 1;
+}
+
+message User {
+    uint32 id = 1;
+}
+
+message Thread {
+    google.protobuf.Timestamp start = 1;
+    string name = 2;
+    int32 tid = 3;
+    int32 namespaced_tid = 4;
+    string syscall = 5;
+    bool compat = 6;
+    optional UserStackTrace user_stack_trace = 7;
+}
+
+message UserStackTrace {
+    repeated uint64 addresses = 1;
+}
+
+message Container {
+    string id = 1;
+    string name = 2; 
+    ContainerImage image = 3;
+    bool started = 4;
+}
+
+message ContainerImage {
+    string id = 1;
+    repeated string repo_digests = 2;
+    string name = 3;
+}
+
+message K8S {
+    Pod pod = 1;
+    K8sNamespace namespace = 2;
```

Comment:
Another nit: the struct is called `K8S` but here you use `K8s` with `s` - don't we want to be consistent about it?